mixer/cmd/build.go

```diff
@@ -0,0 +1,162 @@
+// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Just checking that Apache is what is intended.

Reply:
Yes this is just copying what mixer was initially released with.